src/modules/News/NewsList.tsx

```diff
@@ -0,0 +1,149 @@
+
+  useEffect(() => {
+    getNewsList();
+  }, [getNewsList]);
+
+  const buttons = useMemo(
+    () => (
+      <Box mb={2}>
+        {['ALL', 'INF', 'MFI'].map(name => (
+          <NewsButton
+            name={name}
+            key={name}
+            type="button"
+            text={t('newsPage.' + name.toLowerCase())}
+            className={activeButton == name + '-button' ? 'selected' : ''}
+            onClick={() => clickedButtonHandler(name)}
+          ></NewsButton>
+        ))}
+      </Box>
+    ),
+    [t, activeButton, clickedButtonHandler],
+  );
+
+  const newsTiles: JSX.Element[] = useMemo(
+    () =>
+      newsList.map(({ title, datetime, source, shortBody, photo, _id }) => (
+        <StyledLink to={_id} key={title} data-cy="news-tile-container">
+          <DetailsTile backgroundColor={theme.palette.background.paper}>
+            <Grid container direction="row" justifyContent="flex-start" alignItems="flex-start">
+              <Grid item xs={5} style={{ display: 'flex', alignItems: 'center' }}>
+                <Box
+                  component="img"
+                  sx={{
+                    height: 210,
+                    width: 300,
+                    objectFit: 'cover',
+                    borderRadius: '15px 0px 0px 15px',
+                  }}
+                  src={photo}
+                />
+              </Grid>
+              <Grid
+                item
+                xs={7}
+                style={{
+                  overflow: 'hidden',
+                  textOverflow: 'ellipsis',
+                  display: '-webkit-box',
+                  WebkitLineClamp: '8',
+                  WebkitBoxOrient: 'vertical',
+                }}
+              >
+                <Paragraph color={theme.palette.secondary.dark} fontSize={20}>
+                  {title}
+                </Paragraph>
+                <Paragraph fontWeight={700} fontSize={15} color={theme.palette.primary.main}>
+                  {` • ${moment(datetime).format('DD-MM-YYYY')} • ${source}`}
+                </Paragraph>
+                {shortBody.map(
+                  (paragraph, index) =>
+                    paragraph.trim().length != 0 && (
+                      <Paragraph key={index} fontWeight={400} fontSize={15} color={theme.palette.primary.dark}>
+                        {paragraph}
+                      </Paragraph>
+                    ),
+                )}
+              </Grid>
+            </Grid>
+          </DetailsTile>
+        </StyledLink>
+      )),
+    [
+      newsList,
+      theme.palette.background.paper,
+      theme.palette.secondary.dark,
+      theme.palette.primary.dark,
+      theme.palette.primary.main,
+    ],
+  );
+
+  if (!isLoading && errorMessage) {
+    return (
+      <p style={{ marginTop: '150px' }} data-cy="error-message">
+        {errorMessage}
+      </p>
+    );
+  }
+
+  if (isLoading && !errorMessage) {
+    return (
+      <>
+        {buttons}
+        <ListPageSkeleton />;
```

Comment:
Modify the width of skeleton tiles to match width of the news tiles.

Reply:
fixed